template/web/src/modules/Scrolly/ScrollySection/BackgroundVideo.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { FC, useEffect, useState, useRef } from 'react'
+import { graphql } from 'gatsby'
+import Box from '@/atoms/Box'
+import ReactPlayer from 'react-player'
+import {
+  SanityModuleDefaultFields,
+  SanityVideoEmbed,
+  SanityImageType
+} from 'web/types/graphql-types'
+import { gsap } from 'gsap'
+import { ScrollTrigger } from 'gsap/dist/ScrollTrigger'
+
+gsap.registerPlugin(ScrollTrigger)
```

Comment:
we should not register ScrollTrigger multiple times

Reply:
Datei gibt es nicht mehr